### PR TITLE
Restore the validations of YAMLs

### PR DESCRIPTION
**Pull request by mkolesnik**
This was deleted by PR #33

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

**Review by dfarrell07 — CHANGES_REQUESTED** [review]
The validate target was split into more specific targets, and a general `lint` target actually covering all linting was added. The YAML linting is still being run via https://github.com/submariner-io/releases/blob/master/.github/workflows/linting.yml#L55.

This is a good reminder that I should loop back and remove the old validate target.

**Comment by mkolesnik** [comment]
> The validate target was split into more specific targets, and a general `lint` target actually covering all linting was added. The YAML linting is still being run via https://github.com/submariner-io/releases/blob/master/.github/workflows/linting.yml#L55.
> 
> This is a good reminder that I should loop back and remove the old validate target.

If you can bother to actually see what `validate` does here you'll see it's entirely different.

**Review by mangelajo — APPROVED** [review]

File: .github/workflows/validation.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+on:
+  pull_request:
+
+jobs:
+  yamls:
+    name: Release YAMLs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@master
```
Comment (dfarrell07) [inline .github/workflows/validation.yml:13]:
```suggestion
        uses: actions/checkout@v2
```

**Review by dfarrell07 — CHANGES_REQUESTED** [review]
> > The validate target was split into more specific targets, and a general `lint` target actually covering all linting was added. The YAML linting is still being run via https://github.com/submariner-io/releases/blob/master/.github/workflows/linting.yml#L55.
> > This is a good reminder that I should loop back and remove the old validate target.
> 
> If you can bother to actually see what `validate` does here you'll see it's entirely different.

Ah, well thanks for the friendly clarification Mike. :roll_eyes:

**Review by dfarrell07 — APPROVED** [review]

**Review by tpantelis — DISMISSED** [review]

File: releases/v0.8.0-rc0.yaml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 version: v0.8.0-rc0
 name: 0.8.0-rc0
+release-notes: Notes go here
```
Comment (tpantelis) [inline releases/v0.8.0-rc0.yaml:4]: Why was this added?

Comment (mkolesnik) [reply releases/v0.8.0-rc0.yaml:4]: It's by this commit - https://github.com/submariner-io/releases/pull/57/commits/f288e34befb6045812348337f87e45b78bd604df
